rdf-star-cg-spec.html

```diff
@@ -801,222 +801,115 @@ <h2>SPARQL* Update</h2>
-        </pre>
-
-        <p>Formally, the <a href="#bnode-scope-ex2">second graph</a> is satisfied by an interpretation having:</p>
-        <ul>
-          <li>|IS|: `:alice`→A, `:knows`→K, `dc:creator`→C, `:bob`→B</li>
-          <li>|IT|: `&lt;&lt; :bob :name "Bob" &gt;&gt;`→T1, `&lt;&lt; :bob :workingFor :acme &gt;&gt;`→T2</li>
-          <li>|IEXT|(K) = {(A,Y)}</li>
-          <li>|IEXT|(C) = {(T1, A), (T2, A)}</li>
-        </ul>
-        <p>with Γ: `_:x`→`:bob`, and A: `_:y`→Y. But this interpretation can not satisfy <a href="#bnode-scope-ex1">the first graph</a>, because it would require a <a>grounding function</a> Γ' such that</p>
-        <ul>
-          <li>I(Γ'(`_:x`)) = Y in order to satisfy the first triple,</li>
-          <li>Γ'(`_:x`) = `:bob` in order to satisfy the second and third triple,</li>
-          <li>and both can not be true at the same time in the interpretation above.</li>
-        </ul>
-      </section>
-
-      <section id="interpolation-lemma">
-        <h2>Interpolation lemma</h2>
-
-        <p>The <a data-cite="RDF11-MT#dfn-interpolation">interpolation lemma</a> [[RDF11-MT]] states that an RDF graph G simply entails an RDF graph E if and only if a subgraph of G is an <a data-cite="RDF11-MT#dfn-instance">instance</a> of E. Intuitively, this means that all graphs simply entailed by G can be constructed by:</p>
-
-        <ul>
-          <li>removing triples,</li>
-          <li>replacing a term with a fresh blank node,</li>
-          <li>splitting a blank node into two (or more) fresh blank nodes.</li>
-        </ul>
+      </pre>
 
-        <p>A design goal of the RDF* semantics was to preserve that property.</p>
+      <p>The first `_:x` is interpreted as a <a>blank node</a> (let's call it |x|) acting as an existential variable that can be mapped to virtually anything. The second `_:x` is internally changed to |Q|(|x|), another <a>blank node</a>, which can be mapped only to <a>RDF* terms</a>, as it must be in the range of <var>S*</var>. Both <a>blank nodes</a> |x| and |Q|(|x|) are constrained by the <var>I*</var> property, which ensures that, modulo one level of indirection, they always represent the same thing.</p>
 
-        <p class="issue">We didn't prove it yet...</p>
-      </section>
+      <p>Finally, it is worth noticing that the <a>IRIs</a> <var>D*</var>, <var>S*</var>, <var>P*</var>, <var>O*</var> and <var>I*</var> are not explicitly specified (besides their existence) by this document. With this omission, this specification provides no standard way to convey an <a>RDF* graph</a> using older RDF concrete syntaxes – only the RDF*-specific extensions of those concrete syntaxes can be used for that. Otherwise, it would have been possible to describe incomplete or ill-formed <a>RDF* triples</a> (a problem that exists with <a data-cite="RDF-SCHEMA#ch_collectionvocab">RDF collections</a> for example), thus forcing implementations to deal with pathological cases, and preventing useful optimisations.</p>
```

Comment:
```suggestion
      <p>Finally, it is worth noticing that the <a>IRIs</a> <var>D*</var>, <var>S*</var>, <var>P*</var>, <var>O*</var> and <var>I*</var> are not explicitly specified (beyond their existence) by this document. With this omission, this specification provides no standard way to convey an <a>RDF* graph</a> using older RDF concrete syntaxes &mdash; only the RDF*-specific extensions of those concrete syntaxes can be used for that. Otherwise, it would have been possible to describe incomplete or ill-formed <a>RDF* triples</a> (a problem that exists with <a data-cite="RDF-SCHEMA#ch_collectionvocab">RDF collections</a> for example), thus forcing implementations to deal with pathological cases, and preventing useful optimisations.</p>
```
I think this paragraph needs further work, as I do not understand what's meant by the text following the first sentence, and more, how/why what's said in those latter sentences follow from what's said in the first.

Reply:
Imagine that the spec specified S* = `rdfx:s`, P* = `rdfx:p` ... and defined a specific IRI for prefix `rdfx:`. Then, the following Turtle(*) document:
```
[]  rdfx:s "<http://example.org/s1>^^rdfx:d;
    rdfx:p "<http://example.org/p1>^^rdfx:d;
    rdfx:o "<http://example.org/o1>^^rdfx:d;  
    dct:creator :alice.
```
would be semantically equivalent to the following Turtle* document:
```
<<:s1 :p1 :o1>> dct:creator :alice.
```
and should therefore be parsed as such by compliant RDF* systems.

More tricky, the following Turtle(*) document:
```
[] rdfx:s "<http://example.org/s1>^^rdfx:d;
   dct:creator :alice.
```
would be semantically equivalent to the following Turtle* document:
```
<<:s1 [] [] >> dct:creator :alice.
```
and should therefore be parsed as such by compliant RDF* systems.

The following Turtle(*) document:
```
[] rdfx:s "<http://example.org/s1>^^rdfx:d, "<http://example.org/s2>^^rdfx:d.
```
would be inconsistent, and that also would have to be dealt with.

To sum it up, that would cause too much burden on RDF* systems. So instead of opening the pandora box, we define the mapping on a theoretical level only.

Does that make more sense? If so, how would you suggest to rephrase the above paragraph?